app/imports/startup/client/routes.jsx

```diff
@@ -1,16 +1,54 @@
 import React from 'react';
 import {mount} from 'react-mounter';
 
-import { AppLayout } from '/imports/ui/appLayout.jsx';
-import Content from '/imports/ui/content.jsx';
-import Header from '/imports/ui/header.jsx';
-import Footer from '/imports/ui/footer.jsx';
+import { MainLayout } from '/imports/components/mainLayout.jsx';
+import ApplyContainer from '/imports/containers/applyContainer.jsx';
```

Comment:
Some files export somthing thing by default
```
export default Foo = () => { console.log('foo'); }
export Bar = () => { console.log('bar'); }

```

In this case you could do
`import Foo from 'path/to/file';`
Or
`import { Bar } from 'path/to/file';`

For `Foo` the brackets are optional but not for `Bar`

If no default is defined you need the brackets